cookbooks/aws-parallelcluster-slurm/recipes/config/config_login.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: config_compute
+#
+# Copyright:: 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: rename, find a better name that include login nodes
+setup_munge_compute_node
```

Comment:
setup_munge_client ?

Reply:
mmm ..it's not exactly the client.

I'll leave this change to a dedicated refactoring.